da/celestia/config.go

```diff
@@ -31,7 +32,7 @@ type Config struct {
 	NamespaceIDStr string               `json:"namespace_id,omitempty"`
 	AuthToken      string               `json:"auth_token,omitempty"`
 	Backoff        uretry.BackoffConfig `json:"backoff,omitempty"`
-	RetryAttempts  int                  `json:"retry_attempts,omitempty"`
+	RetryAttempts  *int                 `json:"retry_attempts,omitempty"`
```

Comment:
why u need it as a pointer?

Reply:
its the easiest way i found to differentiate whether the param is not set or it is set to 0. Happy to change it if there is a better way to do it.

Reply:
`const UNDEFINED = -1`
`const FOREVER = 0`